recipes-sota/ostree-kernel-initramfs/ostree-kernel-initramfs_0.0.1.bb

```diff
@@ -0,0 +1,52 @@
+SUMMARY = "Ostree linux kernel, devicetrees and initramfs packager"
+DESCRIPTION = "Ostree linux kernel, devicetrees and initramfs packager"
+SECTION = "kernel"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0;md5=801f80980d171dd6425610833a22dbe6"
+
+# Whilst not a module, this ensures we don't get multilib extended (which would make no sense)
+inherit module-base
+
+PACKAGES = "ostree-kernel ostree-initramfs ostree-devicetrees"
```

Comment:
How do these packages actually get populated? I think I'm missing the connection.

Reply:
These packages are being installed by IMAGE_INSTALL

Reply:
I guess I meant, how do you define what the contents are of each of these packages? I'm not seeing how the files that are getting copied/installed below actually get assigned to one or the other of these packages.

Reply:
I am not sure if I understood you question, but if you meant how the artifacts being packaged, the answer would be by FILES?
```
+FILES_ostree-kernel = "${nonarch_base_libdir}/modules/*/vmlinuz"
+FILES_ostree-initramfs = "${nonarch_base_libdir}/modules/*/initramfs.img"
+FILES_ostree-devicetrees = "${nonarch_base_libdir}/modules/*/dtb/*"
```

Reply:
Ah, perfect, thanks. I was overlooking that bit.